Services/JenkinsDocker/Scripts/upgrade.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+set -x
+azure servicefabric application package copy JenkinsSF fabric:ImageStore > /dev/null 2>&1
+if [ $? -ne 0 ]; then
+    echo "Application copy failed."
+    exit 1
+fi
+azure servicefabric application type register JenkinsSF > /dev/null 2>&1
+if [ $? -ne 0 ]; then
+    echo "Application type registration failed."
+    exit 1
+fi
+version=$(sed -e "s/xmlns/ignore/" JenkinsSF/ApplicationManifest.xml | xmllint --xpath "string(//ApplicationManifest/@ApplicationTypeVersion)" -)
+eval azure servicefabric application upgrade start --application-name fabric:/JenkinsSF --target-application-type-version ${version} --rolling-upgrade-mode Monitored > /dev/null 2>&1
```

Comment:
At some places we are using the application name as fabric:/jenkinsSF and other places fabric:/JenkinsSF. We need to fix this or else things wont work as the name is case sensitive